contrib/wireshark/rmpp.lua

```diff
@@ -0,0 +1,271 @@
+--
+-- Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2023. ALL RIGHTS RESERVED.
+-- See file LICENSE for terms.
+--
+
+-- Usage:
+-- $ wireshark -Xlua_script:rmpp.lua <pcap_file>
+--
+-- or copy the LUA file in the Wireshark plugins directory.
+--   - Help -> About Wireshark -> Folders -> Personal Lua plugins
+
+local rmpp_version_str = {
+    [0] = "Not Set",
+    [1] = "1",
+}
+
+local RMPP_STATUS_SUCCESS    =   0
+local RMPP_STATUS_RESX       =   1
+local RMPP_STATUS_ABORT_MIN  = 118
```

Comment:
why is this definition needed?

Reply:
removed thanks. it is actually used by umad kernel module along with abort max to identifying failure code ranges